apps/consent/app/oidc-cert.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright © 2023 Ory Corp
```

Comment:
this file should be deleted

Reply:
deleted on latest https://github.com/GaloyMoney/galoy/pull/3320